src/safety/SafetyConfig.js

```diff
@@ -75,14 +88,45 @@ const safetyConfig = {
         label: "Very High"
       }]
     },
-      'harsh_breaking_ratio': getRenderer('harsh_breaking_ratio', [0,0.02,0.1,1.5,14],['Low','Medium','High'],"Harsh braking"),
-    'harsh_cornering_ratio': getRenderer('harsh_cornering_ratio', [0,0.02,0.1,1.5,14],['Low','Medium','High'],"Harsh cornering"),
-    'pedestrians_density': getRenderer('pedestrians_density', [0,1,14,20,1.2],['Low','Medium','High'],"Average pedestrian volume"),
-    'bicycles_density': getRenderer('bicycles_density', [0,1,14,20,1.2],['Low','Medium','High'],"Average cyclist volume"),
-    'speeding_ratio': getRenderer('speeding_ratio', [0,0.01,0.1,0.5,15],['Low','Medium','High'],"Above average speed"),
-    'average_speed': getRenderer('average_speed',[28,38,48,58,68],['< 25','50','> 70'],"Average speed"),
-    'pcw': getRenderer('pcw', [0,0.01181361,0.02357791,0.03498221,0.05745235],['Low','Medium','High'],"Pedestrian collision warning (PCW)"),
-    'fcw': getRenderer('fcw', [0,0.01181361,0.04357791,0.5,1],['Low','Medium','High'],"Forward collision warning (FCW)")
+    'harsh_breaking_ratio': getRenderer('harsh_breaking_ratio', 
+      [QUERY_ME8_DATA, QUERY_OEM_DATA], 
+      [[0,0.02,0.1,1.5,14], [0,0.0,0.0,0.0,0.1]],
+      ['Low','Medium','High'], "Harsh braking"),
+
+    'harsh_cornering_ratio': getRenderer('harsh_cornering_ratio', 
+      [QUERY_ME8_DATA, QUERY_OEM_DATA],
+      [[0,0.02,0.1,1.5,14], [0,0.0,0.0,0.0,0.1]],
+      ['Low','Medium','High'], "Harsh cornering"),
+
+    'pedestrians_density': getRenderer('pedestrians_density', 
+      [QUERY_ME8_DATA, QUERY_OEM_DATA],
+      [[0,1,14,20,1.2], [0,1,2,3,4]],
+      ['Low','Medium','High'], "Average pedestrian volume"),
+
+    'bicycles_density': getRenderer('bicycles_density', 
+      [QUERY_ME8_DATA, QUERY_OEM_DATA],
+      [[0,1,14,20,1.2], [0,1,2,3,4]],
+      ['Low','Medium','High'], "Average cyclist volume"),
+
+    'speeding_ratio': getRenderer('speeding_ratio',
+      [QUERY_ME8_DATA, QUERY_OEM_DATA],
+      [[0,0.01,0.1,0.5,15], [0,0.01,0.1,0.2,1]],
+      ['Low','Medium','High'], "Above average speed"),
+
+    'average_speed': getRenderer('average_speed',
+      [QUERY_ME8_DATA, QUERY_OEM_DATA],
+      [[28,38,48,58,68], [9,12,16,20,23]],
+      ['< 25','50','> 70'], "Average speed"),
+
+    'pcw': getRenderer('pcw',
+      [QUERY_ME8_DATA, QUERY_OEM_DATA],
+      [[0,0.01181361,0.02357791,0.03498221,0.05745235], [0.0,0.01,0.02,0.03,0.05]],
+      ['Low','Medium','High'], "Pedestrian collision warning (PCW)"),
+
+    'fcw': getRenderer('fcw',
+      [QUERY_ME8_DATA, QUERY_OEM_DATA],
+      [[0,0.01181361,0.04357791,0.5,1], [0.0,0.01,0.02,0.03,0.05]],
+      ['Low','Medium','High'], "Forward collision warning (FCW)")
```

Comment:
I randomly set up the second sets of color stop points just to demonstrate the visual changes when switching data source.